cf/rds.yaml

```diff
@@ -0,0 +1,123 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Set up RDS Aurora cluster [managed by github.com/hms-dbmi-cellenics/iac]
+
+Parameters:
+  Environment:
+    Type: String
+    Default: development
+    AllowedValues:
+      - development
```

Comment:
similarly to the previous template -- what do you use development for? All other stacks are either in staging or in production

Reply:
hmm, just noticed that I am completely wrong and we have development as an environment everywhere. Very strange

Reply:
I assume that at some point we used to run redis locally, and now we don't and it wasn't removed from here

Reply:
Or it got mistakenly copied over from other files, because it was needed there.
@cosa65 what will be the situation with rds? Will we run a local instance of it similarly to how we run redis?

Reply:
Locally we are setting [up a postgres](https://github.com/hms-dbmi-cellenics/inframock/pull/64) db because rds is an aws-specific service and localstack only offers it in premium version

Reply:
[this discussion](https://github.com/hms-dbmi-cellenics/iac/pull/255#discussion_r815862457) is the same as the current one, let's continue both here

Reply:
Discussed with Martin on a call, let's keep "development" in here -- it is consistent with what we have in other cf templates. If we don't have it here, it will be weird.